website/docs/docs/cloud/git/connect-github.md

```diff
@@ -30,13 +30,17 @@ To connect your <Constant name="cloud" /> account to your GitHub account:
 
 3. Scroll down to **Linked accounts**.
 
-<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-connect-1.png" title="Navigated to Linked Accounts under your profile"/>
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-connect-1.png" width= "80%" title="Navigated to Linked Accounts under your profile"/>
 
 4. In the **Linked accounts** section, set up your GitHub account connection to <Constant name="cloud" /> by clicking **Link** to the right of GitHub. This redirects you to your account on GitHub where you will be asked to install and configure the <Constant name="cloud" /> application. 
 
 5. Select the GitHub organization and repositories <Constant name="cloud" /> should access.
 
-   <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-app-install.png" title="Installing the dbt application into a GitHub organization"/>
+   <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-app-install.png" width="50%" title="Installing the dbt application into a GitHub organization"/>
+
+   :::important Case-sensitive repository names
+   When specifying a GitHub repository in the <Constant name="dbt_platform" /> via the UI, API, or Terraform provider, the repository name must exactly match the case used in the GitHub URL to avoid cloning errors or job failures.
```

Comment:
sugg an alternative since we avoid using [latin abbreviations](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#latin-abbreviations)
```suggestion
   When specifying a GitHub repository in the <Constant name="dbt_platform" /> using the UI, API, or Terraform provider &mdash; the repository name must exactly match the case used in the GitHub URL to avoid cloning errors or job failures.
```

Reply:
not a blocker but not sure if this is the best placement for this callout? since it applies to other types of integrations - it might be better placed just after the prerequistes?